lib/hawkular/metrics/metric_api.rb

```diff
@@ -129,6 +129,19 @@ def get_data(id, starts: nil, ends: nil, bucketDuration: nil, buckets: nil, perc
         resp.is_a?(Array) ? resp : [] # API returns no content (empty Hash) instead of empty array
       end
 
+      # Retrieve raw data for multiple metrics.
+      # This API is experimental and may undergo non-backwards compatible changes on the future
+      # @param ids [Array[String]] metric definition ids
+      # @param starts [Integer] optional timestamp (default now - 8h)
+      # @param ends [Integer] optional timestamp (default now)
+      # @param limit [Integer] optional limit the number of data points returned
+      # @param order [String] optional Data point sort order, based on timestamp (ASC, DESC)
+      # @return [Array[Hash]] named datapoints
+      def raw_data(ids, starts: nil, ends: nil, limit: nil, order: nil)
+        params = { ids: ids, start: starts, end: ends, limit: limit, order: order }
+        @client.http_post("/#{@resource}/raw/query", params)
```

Comment:
@pilhuhn I wonder why retrieval of metrics should be done in POST rather than GET...